package.json

```diff
@@ -8,7 +8,7 @@
     "prepare": "npm run build",
     "release": "git tag $npm_package_version && git push --tags && npm publish",
     "lint:js": "eslint --ignore-path .gitignore --ignore-pattern webpack .",
-    "test": "npm run test:pre && cross-env NODE_ENV=test nyc mocha tests/mocha.js ./tests/**/*.tests.js ./tests/*.tests.js",
+    "test": "npm run test:pre && cross-env NODE_ENV=test nyc mocha tests/mocha.js ./tests/**/**/*.tests.js ./tests/**/*.tests.js ./tests/*.tests.js",
```

Comment:
if i understand this correct, we should use single quotes around `./tests/**/*.tests.js` and cover all 3 cases:
https://github.com/mochajs/mocha/issues/1828#issuecomment-130233858

something like this:
```
"test": "npm run test:pre && cross-env NODE_ENV=test nyc mocha -r tests/mocha.js './tests/**/*.tests.js'",
```